### DISCUSSION TITLE
Hi, having some issues applying for the Global Campus

### DISCUSSION THREAD

**Discussion by AdrianJS2009** [community/community#41657] [GitHub Education] [answered]
Hi, i''ve tried to apply to the Global Campus since i'm now studying on a Public School from Spain. I've tried to submit the apply via "Other (Example Screenshot of school portal)",  sharing a screen from the official online portal school via "iPasen" showing my schedule from Monday to Friday. "iPasen" is the official platform used in Andalucia for public Schools.

When i try to apply i get these 2 errors: 

![errors](https://user-images.githubusercontent.com/114468810/207417859-8f1fe736-f776-4526-b72a-d3915f239b51.png)

And this is the image that i'm trying to use as proof: 

![proof](https://user-images.githubusercontent.com/114468810/207418298-71846893-fc47-4ed0-8061-f86561adbfc7.png)

And since i'm allowed only to upload one screenshot, i've been extensive nf the proof type description:

> I wish i could upload several screens, but you can see here that im logged on the official webpage/online portal that every public school in Andalucia uses, so they have their students registered all in one single place. 
> 
> I'll adjunt the link as to what  "iPasen" is . It's in Spanish so i apologize in advance.
> 
> https://www.juntadeandalucia.es/educacion/portals/web/ipasen/-que-es-ipasen-#:~:text=iPasen%20es%20la%20aplicaci%C3%B3n%20m%C3%B3vil,%2C%20cargos%20directivos%20y%20profesorado).
> 
> You can also see that im logged in (Which means that i have an active scholarship going). On the right corner of the screen, you can glimpse there the name of the school I.E.S Politecnico Jesus Marin 
> 
> https://politecnicomalaga.com/
> 
> I can assist you with any kind of info it it's needed. As a last remark, i also have a private student email given by the school which is "ajimsan2009@g.educaand.es"
> 
> Which goes for A -> Adrian // jim -> Jimenez // san -> Santiago // 2009 -> 20/09 which is the day i was born in 1996.
> 
> It is a student email given to every student with a scholarship in any public school, we have several benefits from it, for example, we have free unlimited space on Google Drive (I can adjunt a image for that also).
> 
> And the g.educaand after the @ goes for google.educaciónandalucia which translate to AndaluciaEducation. In this screen you can see that the gmail has the official logo of the public government of Andalucia: https://i.imgur.com/50oTrLi.png 
> 
> Thank you.
> 

Thanks.

**Comment by samus-aran** [comment, marked as answer]
Hi @AdrianJS2009 👋 

We're happy you're here. Let's look at the rejections reasons.

1. Ensure your profile contains some reference to your evidence. The verification process cross references your GitHub username, email and profile name (and other aspects). Do any of these match on your evidence?

Although Dated School ID is preferred we do support alternative evidence. Try applying again after updating your GitHub profile 👍

**Reply by AdrianJS2009** [reply]
Thank you. I've managed to make it work properly using another Screen showing the Date of the start of my Scholarship alongside with which subjects im enrolled, appreciate the fast answer and your help.

**Comment by AdrianJS2009** [comment]
Hi, 'm sorry to necro this post, but today i got an email to renew the information regarding my student status which is totally fine, the issue is that it demands a specific email syntaxis which may be outdated since all the alumns now we use @g.educaand.es. I'm unable to update info due to this, any help?. Thank you

![image](https://user-images.githubusercontent.com/114468810/226206743-104a289d-dd28-461f-980d-5f1a9def95f5.png)
